src/lib/simulation/maniacalMonkey.ts

```diff
@@ -0,0 +1,20 @@
+import LootTable from 'oldschooljs/dist/structures/LootTable';
+
```

Comment:
Wouldn't this be better in OSJS so you can export it as a monster as well as a loot-table? I don't see why it would require special loot simulation?
This would make it an alternative for monkeys and slayer compatible.

Reply:
yea I agree, it should all be as just a killableMonster. I just didn't wanna wait on the oldschoolJS update. :D